docs/en/integrations/language-clients/java/client-v2.md

```diff
@@ -0,0 +1,539 @@
+---
+sidebar_label: Client V2
+sidebar_position: 2
+keywords: [clickhouse, java, jdbc, client, integrate, r2dbc]
+description: Options for connecting to ClickHouse from Java
+slug: /en/integrations/java/client-v2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+
+# Java Client (V2)
+
+Implementation of a new API. It uses Apache Http Client to communicate with ClickHouse server. We have selected this http client because it has many built-in features and 
+has proven itself in old client implementation. We are planning to support other http client libraries. 
```

Comment:
I would explain why we did it (from an API simplicity perspective)
Why we are saying `We are planning to support other http client libraries`

Reply:
I've reworked this section for both clients. It actually should explain what is this library.